src/Microsoft.PowerShell.Commands.Utility/resources/WebCmdletStrings.resx

```diff
@@ -189,6 +189,9 @@
   <data name="OutFileMissing" xml:space="preserve">
     <value>The cmdlet cannot run because the following parameter is missing: OutFile. Provide a valid OutFile parameter value when using the PassThru parameter, then retry.</value>
   </data>
+  <data name="ResumeOutFileMissing" xml:space="preserve">
+    <value>The cmdlet cannot run because the following parameter is missing: OutFile. Provide a valid OutFile parameter value when using the Resume parameter, then retry.</value>
```

Comment:
Should we parameterize this rather than having two messages almost the same?

Reply:
fixed.